app/views/share/_share.html.erb

```diff
@@ -10,7 +10,7 @@
   <ul>
     
     <% # Twitter share %>
-    <li>
+    <li class="twitter_share_li">
```

Comment:
Can we just call this class `twitter`? We prefer single-word names for our classes -- the `twitter-share-button` class down there is an outlier, required by Twitter itself to make the button work.

Reply:
sure, made that change in e9ae4465782a845fe82682c0d5749cfacaecda15